packages/metrics_center/lib/src/common.dart

```diff
@@ -43,7 +43,15 @@ class MetricPoint extends Equatable {
   final SplayTreeMap<String, String> _tags;
 
   @override
-  List<Object?> get props => <Object?>[value, tags];
+  bool operator ==(Object other) {
+    if (other.runtimeType != runtimeType) {
+      return false;
+    }
+    return other is MetricPoint && other.value == value;
```

Comment:
Can we add the tags in the `==` logic? Different MetricPoint instances may have same `value` but different `tags`.

Reply:
Wow I totally missed the `_tags` field.

With the tags as part of the equality, we can just remove the operator== and hashCode entirely, because no two MetricPoint objects will ever be equal, since they all instantiate a new private instance of an object that cannot be extracted from the class. That makes the PR much simpler!